sections/kubecontext.zsh

```diff
@@ -17,6 +17,7 @@ SPACESHIP_KUBECONTEXT_SUFFIX="${SPACESHIP_KUBECONTEXT_SUFFIX="$SPACESHIP_PROMPT_
 SPACESHIP_KUBECONTEXT_SYMBOL="${SPACESHIP_KUBECONTEXT_SYMBOL="☸️  "}"
 SPACESHIP_KUBECONTEXT_COLOR="${SPACESHIP_KUBECONTEXT_COLOR="cyan"}"
 SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW="${SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW=true}"
+SPACESHIP_KUBECONTEXT_COLORGROUPS=(${SPACESHIP_KUBECONTEXT_COLORGROUPS=()})
```

Comment:
Should this be `COLORGROUPS` or `COLOR_GROUPS` ? 🤔

Reply:
For the record, I don't mind either name, feel free to change if you like.

Reply:
Either would work, I don't mind either. "Colorgroups" and "Color groups" have a slight semantic meaning different IMO but both are good.
I guess that looking at other names, separating the words is the convention so I'll do that.